src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+// body {
+// margin: 0;
+// padding: 0;
+
+// display: flex;
+// justify-content: center;
+// font-family: Roboto, sans-serif;
+// }
+
+.product-card {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+
+  box-sizing: border-box;
+
+  border-radius: 5px;
+  border: $card-border-width solid #f3f3f3;
+  padding: $card-padding;
+
+  transition: transform 300ms;
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    text-decoration: none;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-top: 40px;
+    transition: color 300ms;
+
+    @include blocks-width;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-text-color;
+    margin: 4px 0 0;
+
+    @include blocks-width;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+
+    @include blocks-width;
+
+    &_label {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $second-text-color;
+    }
+
+    &_value {
+      color: $main-text-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+    }
+  }
+
+  &__buy {
+    text-decoration: none;
+    color: white;
+    background-color: #00acdc;
+    height: 40px;
+    border-radius: 5px;
+    margin-top: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    text-align: center;
+    transition-property: color, background-color;
+    transition-duration: 300ms;
+
+    @include blocks-width;
+
+    &:hover {
+      color: #00acdc;
+      background-color: white;
+      border: 1px solid #00acdc;
+    }
```

Comment:
The hover effect on the &__buy element changes its color and background. Ensure that the contrast remains accessible and the text is readable.